installers/ansible/inventory.yaml

```diff
@@ -0,0 +1,30 @@
+---
+    all:
+        hosts:
+            localhost:
+        vars:
+            ansible_connection: local
+            ansible_python_interpreter: "/usr/bin/env python"
+            # ==================
+            # Installation Methods
+            # One of the following blocks must be updated:
+            # - Deploy into Kubernetes
+            # - Deploy into Openshift
+
+            # Deploy into Openshift
+            # ==================
+            # Note: openshift_host can use the format https://URL:PORT
+            # Note: openshift_token can be used for token authentication
+            # ==================
+            # openshift_host: ''
+            # openshift_skip_tls_verify: true
+            # openshift_user: ''
+            # openshift_password: ''
+            # openshift_token: ''
+
+            # Deploy into Kubernetes
+            # ==================
+            # Note: Context name can be found using:
+            #   kubectl config current-context
+            # ==================
+            # kubernetes_context: ''
```

Comment:
Does this need to be in here?

Reply:
It needs to be somewhere. This is how ansible connects to the kube/ocp cluster when you aren't using the deployer. It doesn't have to be in that file but since it isn't used to configure the operator i thought it would be good to separate it into this file.

If someone is installing with ansible they will need to update the connection info here and update configuration parameters in the values.yaml file. Both of these files will then be used to install

Reply:
OK. Please put the Kubernetes block before OpenShift.